src/Sylius/Bundle/UserBundle/Command/AbstractRoleCommand.php

```diff
@@ -112,7 +116,7 @@ protected function getEntityManager(string $userType): ObjectManager
     {
         $class = $this->getUserModelClass($userType);
 
-        return $this->getContainer()->get('doctrine')->getManagerForClass($class);
+        return $this->container->get('doctrine')->getManagerForClass($class);
```

Comment:
@stloyd to avoid BC-break, maybe you can add a getContainer() method returning $this->container.
what do you think?